static/app/components/events/interfaces/frame/stacktraceLink.tsx

```diff
@@ -230,6 +231,34 @@ export function StacktraceLink({frame, event, line, disableSetup}: StacktraceLin
     refetch();
   };
 
+  const filePath =
+    frame.filename && frame.lineNo ? frame.filename + ':' + frame.lineNo : '';
+
+  const {onClick: handleCopyPath} = useCopyToClipboard({
+    text: filePath,
+    successMessage: t('File path copied to clipboard'),
+    errorMessage: t('Failed to copy file path'),
+    onCopy: () => {
+      trackAnalytics('integrations.stacktrace_file_path_copied', {
```

Comment:
Is this specific to integrations? Also if you use this function you would need to also define the corresponding Amplitude label in utils/analytics/*. But if you use the button props you can define the `analyticsEventName` along with the `analyticsEventKey`